pkgs/top-level/all-packages.nix

```diff
@@ -6312,6 +6312,8 @@ in
 
   redo = callPackage ../development/tools/build-managers/redo { };
 
+  reno = pythonPackages.reno;
```

Comment:
is this a Python application, or a library? From you description it seems to be an application. In that case, it should be put elsewhere in Nixpkgs, not in `python-packages.nix`, since that is reserved for Python libraries.